docs/builds/guides/integration/configuration.md

```diff
@@ -140,13 +140,16 @@ You can use the extended toolbar configuration format to access additional optio
 ```js
     toolbar: {
         items: [ 'bold', 'italic', '|', 'undo', 'redo', '-', 'numberedList', 'bulletedList' ],
+        removeItems: [ 'link' ],
         viewportTopOffset: 30,
         shouldNotGroupWhenFull: true
     }
 ```
 
  * **`items`** &ndash; An array of toolbar item names. Most of the components (buttons, dropdowns, etc.) which can be used as toolbar items are described under the {@link features/index Features} tab. A full list is defined in {@link module:ui/componentfactory~ComponentFactory editor.ui.componentFactory} and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`. More details could be found in the {@link framework/guides/creating-simple-plugin Creating a simple plugin} guide.
 
+ * **`removeItems`** &ndash; An array of toolbar item names. With this setting you can modify the default toolbar items of a bundle without the need of recreating it from scratch. If, after removing an item, toolbar will have two or more consecutive separators (`'|'`), the duplicates will be removed automatically.
```

Comment:
```suggestion
 * **`removeItems`** &ndash; An array of toolbar item names. With this setting you can remove items without recreating the entire items array from scratch. It is especially useful when you want to remove a certain plugin from an editor build that has a predefined toolbar. Moreover, if after removing an item, the toolbar will have two or more consecutive separators (`'|'`), the duplicates will be removed automatically.
```

Reply:
*   bundle -> build as this is the term we use more often.
*   Separated the last sentence with something that makes it stand out a bit more because.
*   Made it clear when this option is useful to give a bit more context.